src/chainparams.cpp

```diff
@@ -460,10 +460,10 @@ class CMainParams : public CChainParams {
         };
 
         chainTxData = ChainTxData{
-            1574164758, // * UNIX timestamp of last known number of transactions (Block 1173619)
-            19444536,   // * total number of transactions between genesis and that timestamp
+            1601017518, // * UNIX timestamp of last known number of transactions (Block 1344000)
+            28914483,   // * total number of transactions between genesis and that timestamp
                         //   (the tx=... number in the SetBestChain debug.log lines)
-            0.2         // * estimated number of transactions per second after that timestamp
+            0.3         // * estimated number of transactions per second after that timestamp
```

Comment:
How does `getchaintxstats` calculate txrate? My node shows 0.309... But I wonder what time / block range it is calculated on

Reply:
It's calculated for the past month. My node also shows some uneven number but I thought it should be fine to take that rate and to round it a bit since it's an estimate anyway.

Reply:
Agree